frontend/src/scenes/urls.ts

```diff
@@ -7,12 +7,20 @@ export const urls = {
     notFound: () => '404',
     dashboards: () => '/dashboard',
     dashboard: (id: string | number) => `/dashboard/${id}`,
-    createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
-    action: (id: string | number) => `/action/${id}`,
-    actions: () => '/events/actions',
-    eventStats: () => '/events/stats',
-    eventPropertyStats: () => '/events/properties',
-    events: () => '/events',
+    // TODO: Deprecate below with `collaborations-taxonomy` FF
+    LEGACY_createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
+    LEGACY_action: (id: string | number) => `/action/${id}`,
+    LEGACY_actions: () => '/events/actions',
+    LEGACY_eventStats: () => '/events/stats',
+    LEGACY_eventPropertyStats: () => '/events/properties',
+    LEGACY_events: () => '/events',
+    // TODO: Deprecate above with `collaborations-taxonomy` FF
+    events: () => '/events_v2', // live events
+    taxonomy: () => '/taxonomy',
+    taxonomyEventStats: () => '/taxonomy/events/stats',
+    taxonomyEventStatsView: (id: string | number) => `/taxonomy/events/stats/${id}`,
+    taxonomyCalculatedEvents: () => '/taxonomy/calculated_events',
```

Comment:
What is a calculated event? If we're renaming actions I think we should tackle this separately as it'll introduce significant confusion for existing users.

Reply:
Yes Rick and I were talking about how actions is a misleading term that makes it hard to distinguish from what an event is. But totally agree, let's tackle this renaming thing in a separate initiative.